### ISSUE TITLE
GL_NUM_EXTENSIONS is not available on GLES2

### ISSUE THREAD

**Issue by loganmc10** [gonetz/GLideN64#1092]
This commit:
https://github.com/gonetz/GLideN64/commit/1557d519afa37f1988dfebb09139f990d8347d5b

Changed the way that OGLVideo::isExtensionSupported works.

It uses GL_NUM_EXTENSIONS instead of GL_EXTENSIONS. However, GL_NUM_EXTENSIONS is only available on GLES3+

@fzurita have you run into this? I assume mupen64plus-ae would have issues in the GLES2 version.

**Comment by fzurita** [comment]
Interesting, this is compiling fine right now. But it sounds like an ifdef is in order.

**Comment by loganmc10** [comment]
On the Raspberry Pi I get this:

```
/home/alarm/GLideN64/src/OpenGL.cpp: In static member function 'static bool OGLVideo::isExtensionSupported(const char*)':
/home/alarm/GLideN64/src/OpenGL.cpp:116:16: error: 'GL_NUM_EXTENSIONS' was not declared in this scope
  glGetIntegerv(GL_NUM_EXTENSIONS, &count);
                ^~~~~~~~~~~~~~~~~
/home/alarm/GLideN64/src/OpenGL.cpp:118:64: error: 'glGetStringi' was not declared in this scope
   const char* name = (const char*)glGetStringi(GL_EXTENSIONS, i);
```

So it looks like ```glGetStringi``` is also a GLES3+ thing.

**Comment by fzurita** [comment]
Duh, I just realized I haven't pull that change in yet. I will probably run into the same issue you are having.

**Comment by gonetz** [comment]
@loganmc10 you're right. I've fixed that: 871c42e8c63
